libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnChooseEntity.schema

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/master/schemas/component.schema",
+    "$role": "union(Microsoft.ITriggerCondition)",
+    "title": "ChooseEntity Event",
+    "description": "This defines the actions to take when a ChooseEntity event is raised",
```

Comment:
Can you update the language around this to match the other Onxxx definitions?
    "title": "On choose entity",
    "description": "Actions to perform when there is ambiguity between multiple entities.",